test/Dialect/TorchConversion/canonicalize.mlir

```diff
@@ -0,0 +1,31 @@
+// RUN: torch-mlir-opt %s -canonicalize | FileCheck %s
+
+// CHECK-LABEL:  func.func @torch_c.from_i64_to_i64() -> !torch.vtensor<[4],si64> {
+// CHECK:            %[[CST:.*]] = arith.constant dense<[5, 1, 4, 2]> : tensor<4xi64>
+// CHECK:            %[[T0:.*]] = torch_c.from_builtin_tensor %cst : tensor<4xi64> -> !torch.vtensor<[4],si64>
+// CHECK:            return %[[T0:.*]] !torch.vtensor<[4],si64>
+func.func @torch_c.from_i64_to_i64() -> !torch.vtensor<[4],si64> {
```

Comment:
Can you add unit tests for each individual fold? Having one big function like this makes it hard to debug failures later.

See this file for examples: https://github.com/llvm/torch-mlir/blob/main/test/Dialect/Torch/canonicalize.mlir

Reply:
done